common-content/en/module/how-our-curriculum-works/importance-of-prep/index.md

```diff
@@ -0,0 +1,12 @@
++++
+title="🧑🏾‍💻 Importance of prep"
+headless="true"
+time= 30
+[objectives]
+    1="Explain the importance of prep"
++++
+
+In a flipped learning model, learners are expected to **prepare** before they meet up as a community.
```

Comment:
Can you clean up this habit of putting a single line return so it's not a paragraph break or a complete paragraph? Pick a lane! :p

Reply:
Apolgies. I shall! 🧹